### PR TITLE
Ovs upgrade

### PR DESCRIPTION
**Pull request by Baisang**
Upgrade OVS to 2.7 and make `ovn-controller` not restart every time

Both commits were tested using quilt-tester locally once, all tests passed

### CODE REVIEW

**Comment by kklin** [comment]
Did you test changing ovn-remote?

**Comment by Baisang** [comment]
@kklin not explicitly, what would be a good way to do that?

**Comment by kklin** [comment]
Not sure, I'm not sure there's a _good_ way. Ideally, you'd trigger a leader change by booting two masters, and killing the leader or something. But I bet other code around leader failover is broken.

(BTW, I'm fine merging it for now if we can't figure out a way to test it. Fixing leader failover is an entire project in itself)

File: minion/supervisor/worker.go [hunk 1]
```diff
@@ -69,7 +69,6 @@ func runWorkerOnce() {
 	}
 
 	oldEtcdIPs = etcdIPs
-	oldIP = IP
```
Comment (mmussomele) [inline minion/supervisor/worker.go]: Since machines don't change roles anymore, isn't this assignment ineffectual now? A bit of a nit, but it feels like it deserves it's own patch

Comment (Baisang) [inline minion/supervisor/worker.go]: This was a line that was ineffectual I put in from a previous patch, was just going to trim it with this one. We still need some form of 'state variables' in supervisor currently as it doesn't sync with the state of the real world yet.

**Comment by Baisang** [comment]
@kklin , I tested booting with 2 masters and killed a master on aws. quilt then tried to boot another master, an election was had, and then it removed and replaced/restarted all containers. Everything seemed to work after that.

Let me know if you want the log outputs or anything

**Comment by kklin** [comment]
@Baisang nah, that sounds fine to me

**Comment by kklin** [comment]
Actually, shouldn't ovn-controller never restart on workers now?

**Comment by Baisang** [comment]
@kklin I believe so, unless the container dies for some reason

**Comment by kklin** [comment]
Oh, when you said "replaced/restarted all containers" do you mean the application containers? So not ovn-controller?

**Comment by Baisang** [comment]
yes, i meant the application containers. I didn't check if ovn-controller restarted, I can do that if you'd like

File: minion/supervisor/worker.go (hunk 1 above)
Comment (kklin) [inline minion/supervisor/worker.go]: I think you should put it in its own patch. Then in the commit description you could describe why the line is ineffectual, and thus can be removed. It's helpful to do it in a separate commit because if it ends up that the line _is_ necessary, it'll be easier to pinpoint the offending change.

**Comment by kklin** [comment]
I think it's alright, we'll have to do more thorough failover testing soon anyways.

**Comment by Baisang** [comment]
@kklin moved the ineffectual assignment to another patch
